timer/src/main/java/com/toggl/timer/di/TimerModule.kt

```diff
@@ -91,26 +95,41 @@ class TimerModule {
     @Provides
     @Singleton
     internal fun timerReducer(
+        timeEntryReducer: TimeEntryReducer,
         timeEntriesLogReducer: TimeEntriesLogReducer,
         startEditReducer: StartEditReducer,
         runningTimeEntryReducer: RunningTimeEntryReducer,
         projectReducer: ProjectReducer
     ): TimerReducer {
 
         return combine(
-            timeEntriesLogReducer.pullback(
+            timeEntriesLogReducer.decorateWith(timeEntryReducer,
+                mapToLocalState = { TimeEntryState(it.timeEntries) },
+                mapToLocalAction = { TimeEntryAction.fromTimeEntryActionHolder(it) },
+                mapToGlobalState = { globalState, localState -> globalState.copy(timeEntries = localState.timeEntries) }
```

Comment:
Do you think properly named methods here would be helpful at all? Are we ever going to reuse them?
```suggestion
                mapToLocalState = TimeEntriesLogState.Companion::toTimeEntriesState,
                mapToLocalAction =TimeEntriesLogAction.Companion::toTimeEntriesAction,
                mapToGlobalState = TimeEntriesLogState.Companion::fromTimeEntriesState }
```

Reply:
I don't think this would make much difference :-(